app/src/main/java/org/sopt/sample/home/adapter/ColorAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package org.sopt.sample.home.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.sample.databinding.ColorItemBinding
+import org.sopt.sample.databinding.HeaderItemBinding
+import org.sopt.sample.home.data.Color
+
+class ColorAdapter(context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private val inflater by lazy { LayoutInflater.from(context) }
+    private var colorList: List<Color> = emptyList()
+
+    class ColorViewHolder(
+        private val binding: ColorItemBinding,
+    ) : RecyclerView.ViewHolder(binding.root) {
+        fun onBind(data: Color) {
+            binding.ivBlue.setImageDrawable(binding.root.context.getDrawable(data.image))
```

Comment:
```suggestion
            binding.ivBlue.setImageResource(data.image)
```
`setImageResource ` 사용하면 좋을 것 같습니다

Reply:
이미지를 삭제해서 반영은 못했지만 다음부터는 `setImageResource` 써보도록 할게요!